unittests/array_bug_tester.py

```diff
@@ -55,6 +55,65 @@ def test4(self):
             '::xyz[2][3]' == aaaa_type.decl_string,
             aaaa_type.decl_string)
 
+    def test5(self):
+        code = 'char const arr[4] = {};'
+        src_reader = parser.source_reader_t(self.config)
+        global_ns = declarations.get_global_namespace(
+            src_reader.read_string(code))
+        arr_type = global_ns.variable('arr').type
+        if self.config.xml_generator == "gccxml":
+            self.assertTrue(
+                'char[4] const' == arr_type.decl_string,
```

Comment:
Does it make sense to you that gccxml is putting the cv-qualifiers on the right of the array declarator in `decl_string`?